modules/applications-create-using-cli-template.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * applications/creating-new-applications.adoc
+
+[id="applications-create-using-cli-template-{context}"]
+= Creating an application from a template
+
+You can create an application from a previously stored template or from a
+template file, by specifying the name of the template as an argument. For
+example, you can store a sample application template and use it to create an
+application.
+
+.Procedure
```

Comment:
Same as the others - I think it would make more sense as a reference than a procedure.